src/utils/index.ts

```diff
@@ -28,13 +27,18 @@ export const deleteCookieOnServerSide = (ctx: NextPageContext, cookieName: strin
     }
 };
 
+export const deleteCookieOnServerSideFromDocument = (cookie: string): void => {
+    document.cookie = `${cookie}=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;`;
+};
+
 export const deleteAllCookiesOnServerSide = (ctx: NextPageContext): void => {
     if (ctx.req && ctx.res) {
-        const cookies = new Cookies(ctx.req, ctx.res);
-        const host = ctx?.req?.headers?.host;
-        const domain = host ? host.split(':')[0] : '';
-
-        ALL_COOKIES.map(cookieName => cookies.set(cookieName, '', { overwrite: true, maxAge: 0, domain, path: '/' }));
+        const cookies = document.cookie.split('; ');
```

Comment:
Looking at the nookies library this could be a lot easier using that:

```
const cookies = parseCookies(ctx);

        Object.keys(cookies).forEach(cookie => {
            if (cookie !== OPERATOR_COOKIE) {
                destroyCookie(ctx, cookie);
            }
        });
```